lib/queuePopulator/LogReader.js

```diff
@@ -355,7 +355,22 @@ class LogReader {
         });
     }
 
+    _isSupportedOperation(record) {
+        // List of methods defined in Metadata/lib/protocol.json.
```

Comment:
Can you specify that we checks if the given record’s method is supported by this LogReader.
And currently, only methods with an ID less than 10 are supported. 
This is intentional so that any future methods (>= 10) that we
do not explicitly support will be rejected by default.